src/packets/ClientProtocol/ClientProtocol_1080/shared.ts

```diff
@@ -1211,6 +1211,41 @@ export function packItemWeaponData(obj: any) {
   return DataSchema.pack(unknownData1Schema, obj).data;
 }
 
+function packFullNPCRemoteWeaponsData(obj: any) {
+  const remoteWeaponsSchema = [
+    {
+      name: "remoteWeapons",
+      type: "byteswithlength",
+      defaultValue: {},
+      fields: [
+        {
+          name: "data",
+          type: "array",
+          defaultValue: [],
+          fields: [
+            { name: "guid", type: "uint64string", defaultValue: "" },
+            ...remoteWeaponSchema,
+          ],
+        },
+        {
+          name: "remoteWeaponExtra",
+          type: "array",
+          defaultValue: {},
+          fields: [
+            { name: "guid", type: "uint64string", defaultValue: "" },
+            ...remoteWeaponExtraSchema,
+          ],
+        },
+      ],
+    },
+  ];
+
+  if (!obj["isVehicle"]) {
```

Comment:
Why not !obj.isVehicle ?
```suggestion
  if (!obj.isVehicle) {
```

Reply:
Idk